src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -198,7 +210,17 @@ private static boolean isTosca(Map<?,?> obj) {
                 String templateId = templateE.getKey();
                 NodeTemplate template = templateE.getValue();
 
-                EntitySpec<VanillaSoftwareProcess> thisNode = new ToscaComputeToVanillaConverter(mgmt).toSpec(templateId, template);
+                EntitySpec<? extends Entity> thisNode = null;
+                try {
+                    thisNode = EntitySpec.create((Class<Entity>) Class.forName(template.getType()));
```

Comment:
if this is useful as a halfway house just add a java comment to the above effect and then we can merge this